src/middlewared/middlewared/plugins/device_/device_info.py

```diff
@@ -291,19 +291,3 @@ def get_gpus(self):
         for gpu in gpus:
             gpu['available_to_host'] = gpu['addr']['pci_slot'] not in to_isolate_gpus
         return gpus
-
-    @accepts()
-    @returns(Dict(
-        'gpu_pci_id_choices',
-        additional_attrs=True,
-        description='Returns PCI id(s) of GPU(s) located in the system',
-        example={'Red Hat, Inc. QXL paravirtual graphic card': '0000:00:02.0'}
-    ))
-    async def gpu_pci_ids_choices(self):
```

Comment:
UI should be using this as well iirc, @Qubad786 can you confirm please ?

Reply:
This should be safe to remove